Automattic/wp-calypso
# Make i5 variation default pricing page

### Changes proposed in this Pull Request

This PR makes the i5 variation of the pricing page the default page.

### Implementation notes

Changelog:
- Moved and renamed `JetpackFreeCard` to `client/components/jetpack/card/jetpack-free-card/`
- Moved and renamed `JetpackProductCard` to `client/components/jetpack/card/jetpack-product-card/`
- Removed unused `jetpack-product-slide-out-card` component
- Removed i5 specific CSS selectors
- Moved and renamed `JetpackFAQ` to `client/my-sites/plans/jetpack-plans/faq/`
- Made i5 copy default
- Removed declared iterations
- Moved and renamed `PlansFilterBar` to `client/my-sites/plans/jetpack-plans/plans-filter-bar/`
- Moved and renamed `ProductCard` to `client/my-sites/plans/jetpack-plans/product-card/`
- Moved and renamed `ProductGrid` to `client/my-sites/plans/jetpack-plans/product-grid/`

### Testing instructions

Download the PR and run both cloud or Calypso concurrently.

For each platform:
- Visit the pricing page locally
- Open a new tab and visit the pricing page in staging or production
- Check that both pages look and behave similarly
- Verify there's no error in the console
